geode-benchmarks/src/main/java/org/apache/geode/benchmark/parameters/HeapParameters.java

```diff
@@ -28,7 +28,7 @@
   public static void configure(final TestConfig testConfig) {
     final String heap = System.getProperty("withHeap", "8g");
     logger.info("Configuring heap parameters {}.", heap);
-    configureAll(testConfig, "-Xmx" + heap, "-Xms" + heap);
+    configureGeodeProductJvms(testConfig, "-Xmx" + heap, "-Xms" + heap);
```

Comment:
I wonder if the work "product" is a bit redundant.

Reply:
This name changed a number of times. Once when we were pairing and then maybe a couple times after. 

Before this PR the method configured all the JVMs. For the first time, with this PR, there are JVMs that do not run Geode (product) software. Those JVMs (`PROXY` ones) do, in a sense, run Geode code—Geode _benchmark_ code.

Since the software in this repo is Geode software, I feel like it's valuable to have the extra qualifier "Product" here to make it clear the the JVMs in question are for running the product, not benchmark code.

Reply:
I still feel like the "product" qualifier is redundant but it isn't hold up this PR.